dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -321,6 +321,30 @@ public Tool updateContainer(@ApiParam(hidden = true) @Auth User user,
 
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{toolId}/defaultVersion")
+    @ApiOperation(value = "Update the default version of the given tool.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Tool.class)
```

Comment:
set nickname in `ApiOperation` to set a unique operation id from the other endpoint

Reply:
Ah. In this case its not really necessary. 
Two different endpoints on two different resources (or rather swagger tags) can share an operationId with no worries.

Reply:
swagger editor complains otherwise (although it already complains about position too).  but i suppose we don't need to appease it

Reply:
Interesting, looking more closely, it looks like something swagger-codegen is fine with, but you're right about validation. Seems like people have had this discussion before. 

https://github.com/go-swagger/go-swagger/issues/1143
https://github.com/OAI/OpenAPI-Specification/issues/381#issuecomment-198255252